index.html

```diff
@@ -0,0 +1,271 @@
+        <hr />
+        <div class="program-container">
+          <div class="program-div">
+            <div class="svg-logo">
+              <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64">
+                <g id="Teamwork">
+                  <path
+                    d="M22.2837,13.3631l2.9538,2.1461a1.9827,1.9827,0,0,1,.72,2.2168L24.83,21.1986a1.9827,1.9827,0,0,0,3.051,2.2166l2.9541-2.1461a1.9823,1.9823,0,0,1,2.3306,0l2.9541,2.1461a1.9827,1.9827,0,0,0,3.051-2.2166L38.042,17.726a1.9828,1.9828,0,0,1,.72-2.2168l2.9539-2.1461a1.9827,1.9827,0,0,0-1.1655-3.5867H36.9a1.9831,1.9831,0,0,1-1.8857-1.37L33.8857,4.9338a1.9826,1.9826,0,0,0-3.7712,0L28.9861,8.4064A1.9829,1.9829,0,0,1,27.1,9.7764H23.4492A1.9827,1.9827,0,0,0,22.2837,13.3631Z"
+                  />
+                  <path
+                    d="M37.4387,44.1386a1.1737,1.1737,0,0,1-.71-1.0786,1.1966,1.1966,0,0,1,.3718-.8779c.0725-.0691.1419-.1372.1939-.192a7.3474,7.3474,0,0,0,.9675-1.2707,7.3009,7.3009,0,0,0-7.637-10.9231,7.134,7.134,0,0,0-5.758,5.5937A7.3,7.3,0,0,0,26.88,42.1668a1.2386,1.2386,0,0,1,.3862.8793v.0061a1.1751,1.1751,0,0,1-.71,1.09,12.0152,12.0152,0,0,0-6.46,6.4014,15.2866,15.2866,0,0,0-1.274,6.2953v1.56A2.547,2.547,0,0,0,21.37,60.9459h21.261a2.5468,2.5468,0,0,0,2.5466-2.5468V56.8234a15.2521,15.2521,0,0,0-1.2761-6.292A12.0124,12.0124,0,0,0,37.4387,44.1386Z"
+                  />
+                  <path
+                    d="M60.8821,47.0135A12.0119,12.0119,0,0,0,54.42,40.6208a1.1745,1.1745,0,0,1-.71-1.0786,1.1973,1.1973,0,0,1,.3719-.878c.0725-.0692.1421-.1372.1938-.1921a7.3327,7.3327,0,0,0,.9676-1.27,7.3007,7.3007,0,0,0-7.6368-10.9232,7.1333,7.1333,0,0,0-5.758,5.5937,7.3,7.3,0,0,0,2.0129,6.7768,1.2373,1.2373,0,0,1,.3862.8792v.0063a1.1745,1.1745,0,0,1-.71,1.09,11.8293,11.8293,0,0,0-3.1953,1.9487,14.7374,14.7374,0,0,1,6.0188,6.8459,17.8408,17.8408,0,0,1,1.5161,7.405v.6043H59.6116a2.5468,2.5468,0,0,0,2.5468-2.5466V53.3056A15.25,15.25,0,0,0,60.8821,47.0135Z"
+                  />
+                  <path
+                    d="M23.66,42.5729a11.8205,11.8205,0,0,0-3.2026-1.9521,1.1744,1.1744,0,0,1-.71-1.0786,1.1968,1.1968,0,0,1,.3721-.878c.0722-.0692.1418-.1372.1938-.1921a7.341,7.341,0,0,0,.9673-1.27,7.3006,7.3006,0,0,0-7.6367-10.9232,7.1338,7.1338,0,0,0-5.7581,5.5937,7.3007,7.3007,0,0,0,2.013,6.7768,1.2369,1.2369,0,0,1,.3862.8792v.0063a1.1747,1.1747,0,0,1-.71,1.09,12.0156,12.0156,0,0,0-6.46,6.4013,15.2875,15.2875,0,0,0-1.2742,6.2953l0,1.5607a2.5465,2.5465,0,0,0,2.5466,2.5466H16.1228v-.5893a17.8839,17.8839,0,0,1,1.5127-7.4058A14.74,14.74,0,0,1,23.66,42.5729Z"
+                  />
+                </g>
+              </svg>
+            </div>
+            <div class="program-tittle">
+              <h3>Partnership</h3>
+            </div>
+            <div class="program-p">
+              Lorem ipsum, dolor sit amet consectetur adipisicing elit.
+              Distinctio, tempora.
+            </div>
+          </div>
```

Comment:
- [ ] I think it would be a good idea if you add some space between your elements, also don't forget that the paragraph it's in the `<p>` tag. Please kindly fix that for all the paragraphs 🙃 😉 

Your design
<img width="377" alt="Screen Shot 2023-01-04 at 00 16 59" src="https://user-images.githubusercontent.com/74501523/210442974-c796db0f-5168-4d57-823f-65b842485948.png">

Original design
<img width="452" alt="Screen Shot 2023-01-04 at 00 17 27" src="https://user-images.githubusercontent.com/74501523/210442991-cf550d9c-05a6-4a4b-9df5-943d7664d399.png">

Reply:
- [ Optional ] As suggested by the previous reviewer to a make your HTML code semantic correct I suggested you to use `p` tag for  "Lorem ipsum, dolor sit amet consectetur adipisicing elit."